web/src/components/storage/ProposalDeviceSection.jsx

```diff
@@ -397,12 +397,11 @@ export default function ProposalDeviceSection({
   isLoading = false,
   onChange = noop
 }) {
-  // FIXME: we should work with devices objects ASAP
-  const { bootDevice } = settings;
+  const targetDevice = settings.targetDevice;
```

Comment:
NP: although the FIXME comment has been drop, I still thinking we should pass the targetDevice, not only its name, to the "BootdeviceSelectorDialog" or whatever component we're going to create.

Reply:
Ok, let's do it as part of the other PR, https://github.com/openSUSE/agama/pull/1106.